### DISCUSSION TITLE
Guidance on Determining Threshold for Filtering PANDA Output TRN

### DISCUSSION THREAD

**Discussion by TheChenSu** [netZoo/netZooPy#304] [Q&A]
Excellent work netZooPy team!

I have recently conducted a comparative analysis using PANDA, with quantile normalized expression data and binary information from ChIP-seq data as inputs. In the resulting GRN, I observed a bimodal distribution. I am seeking guidance on properly determining a threshold for filtering the TF-gene edges relevant to our specific biological context. Given the distribution of scores, we suspect that a score > 0 might not be the most suitable threshold as it may lead to an excessive number of false positive edges. In this regard, could you recommend a suitable threshold for the confidence scores to filter out the most significant edges? I understand that this may vary depending on the nature of the data and the specific project objectives, but any general guidance or best practices would be greatly appreciated. Thanks.

<img width="618" alt="image001" src="https://github.com/netZoo/netZooPy/assets/43080521/f282c4d6-cc9d-41eb-b4f4-313a8643e955">

**Comment by violafanfani** [comment]
Ehi @TheChenSu! This is a long debated question even for us. You are right, the zero threshold would not have any formal/theoretical justification, so I would recommend staying away from it. In general, the authors of the paper have used two main approaches to deal with these scores: 1) keep the scores continuous without thresholding, 2) Threshold at the intersection between the two peaks (which in this case looks close to -0.5). 
Let me know if you want to know more, I will try to connect you with the authors who might be better suited to answer this question.
